src/App.scss

```diff
@@ -59,6 +59,7 @@ body {
 }
 
 .main-container {
+	margin-top: 70px; //To account for height of Navbar, which uses fixed positioning
```

Comment:
This is a small refactor I did.

Previously the decorative Banner was responsible for having extra margins to account for the thickness of the nav-bar (which is stuck using fixed positioning).

This refactor moves the responsibility to the main-container (which is a high level parent of all pages). This makes adding new elements more intuitive, as all content should be wrapped within the main-container. Previously adding new elements would appear at the top of the page, but the content would be hidden behind the fixed positioned nav-bar